gsadjust/GSadjust.py

```diff
@@ -590,66 +605,72 @@ def workspace_save(self):
         """
         if self.label_deltas_update_required.set is True:
             self.msg = show_message(
-                'Workspace cannot be saved while the Relative-gravity differences table on the Network ' +
-                'Adjustment tab is not up to date.', 'Workspace save error')
-            return False
+                'Workspace cannot be saved while the Relative-gravity differences table on the Network '
```

Comment:
Python automagically (and quite annoyingly sometimes) concatenates adjacent strings. If you have braces around them the newlines are taken care of (as for the imports) and you don't need continuation/concat.